charts/lakefs/templates/service.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: {{ include "lakefs.fullname" . }}
   labels:
     {{- include "lakefs.labels" . | nindent 4 }}
+  annotations:
```

Comment:
Hi @ion-elgreco first, THANK YOU for contributing! 

The change make sense, couple of change request: 
1. Notice the CI is failing, please bump the Chart `version` in `Chart.yaml`. **If** after that you see more errors in the CI make sure to fix or LMK if it's not clear why. (every merge release a new chart version automatically). 
2. in `values.yaml` please add a default under the `service` object, i.e 

```yaml
service:
  annotations: {}
  # existing values
  type: ClusterIP
  port: 80
```

Reply:
@Isan-Rivkin done!